samples/caf_sensor_manager/src/main.c

```diff
@@ -3,20 +3,83 @@
  *
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
+
+#include <zephyr.h>
+#include <device.h>
 #include <app_event_manager.h>
+#include <logging/log.h>
+
+#if CONFIG_BOARD_ENABLE_CPUNET
+#include <ipc/ipc_service.h>
+#include <event_manager_proxy.h>
+#endif
+
 
 #define MODULE main
+
 #include <caf/events/module_state_event.h>
+#include <config_event.h>
+#include <caf/events/sensor_data_aggregator_event.h>
 
-#include <logging/log.h>
 LOG_MODULE_REGISTER(MODULE);
 
-
 void main(void)
 {
-	if (app_event_manager_init()) {
-		LOG_ERR("Event manager not initialized");
-	} else {
-		module_set_state(MODULE_STATE_READY);
+	int ret;
+
+	ret = app_event_manager_init();
+	if (ret) {
+		LOG_ERR("Event Manager not initialized, err: %d", ret);
+		__ASSERT_NO_MSG(false);
+		return;
+	}
+	LOG_INF("Event manager initialized");
+
+#if CONFIG_BOARD_ENABLE_CPUNET
+
+	const struct device *ipc_instance  = DEVICE_DT_GET(DT_NODELABEL(ipc0));
+
+	ret = event_manager_proxy_add_remote(ipc_instance);
+	if (ret) {
+		LOG_ERR("Cannot add remote: %d", ret);
+		__ASSERT_NO_MSG(false);
+		return;
+	}
+	LOG_INF("Event proxy remote added");
+
+	/* Listen to the events from the remote */
+	ret = EVENT_MANAGER_PROXY_SUBSCRIBE(ipc_instance,
+					sensor_data_aggregator_release_buffer_event);
+	if (ret) {
+		LOG_ERR("Cannot register to remote sensor_data_aggregator_release_buffer_event: %d", ret);
+		__ASSERT_NO_MSG(false);
+		return;
+	}
+	LOG_INF("Event proxy sensor_data_aggregator_release_buffer_event registered");
+
+	ret = event_manager_proxy_start();
+	if (ret) {
+		LOG_ERR("Cannot start event manager proxy: %d", ret);
+		__ASSERT_NO_MSG(false);
+		return;
+	}
+	LOG_INF("Event manager proxy started");
+
+	/* Wait for all the remote cores to finish event proxy initialization */
+	ret = event_manager_proxy_wait_for_remotes(K_MSEC(CONFIG_SAMPLE_SENSOR_MANAGER_REMOTE_WAIT_TO));
+	if (ret) {
+		LOG_ERR("Error when waiting for remote: %d", ret);
+		__ASSERT_NO_MSG(false);
+		return;
 	}
+	LOG_INF("All remotes ready");
+
+	LOG_INF("Initialization finished successfully");
+#endif /* CONFIG_BOARD_ENABLE_CPUNET*/
+	module_set_state(MODULE_STATE_READY);
+	struct config_event *event = new_config_event();
```

Comment:
we do not need a new event here - please react on module state

Reply:
Same as above.